codecov.yml

```diff
@@ -9,68 +9,68 @@ component_management:
     - component_id: module_leverage_lp_transactions # this is an identifier that should not be changed
       name: leverage_lp_transactions # this is a display name, and can be changed freely
       paths:
-        - ./x/leveragelp/msg_server*
+        - ./x/leveragelp/**/msg_server*
     - component_id: module_accountedpool_transactions
       name: accountedpool_transactions
       paths:
-        - ./x/accountedpool/msg_server*
+        - ./x/accountedpool/**/msg_server*
```

Comment:
does it mean we are including msg_server files only  ? can we not include all the files ?

Reply:
I'm creating different views so we can see coverage from different point of views:
- Queries by modules.
- Transactions by modules.
- Overall transactions.
- Overall queries.

Reply:
got it, makes sense
I think we can also include `abci.go`,  begin block and end block for coverage view

Reply:
Would you prefer to see that coverage isolated or rather include it with the transactions coverage? @amityadav0

Reply:
Isolated would be nice, as that part is different than transactions. Thanks